packages/components/pie-webc-core/src/test/functions/defineCustomElement.spec.ts

```diff
@@ -3,7 +3,7 @@ import { LitElement } from 'lit';
 import {
     vi, expect, it,
 } from 'vitest';
-import { defineCustomElement } from '../defineCustomElement';
+import { defineCustomElement } from '../../functions/defineCustomElement';
```

Comment:
nit, can be addressed in a follow-up PR:

do you think the folder name should be `helpers` to stay consistent with the naming convention in pie-docs and other places?

Reply:
Possibly! Happy to discuss but I think both are quite ambiguous 😂 

Will chat with you offline and fix forward if that's cool 👍